frontend/workflows-lib/stories/WorkflowAccordion.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import WorkflowAccordion from "../lib/components/workflow/WorkflowAccordian";
+import { WorkflowStatus } from "../lib/types";
+
+const fakeTasks = [
```

Comment:
Could we move these to a common TS / JSON file?

Reply:
That could be a separate PR? we could move it for all stories?